static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js

```diff
@@ -32,72 +24,54 @@ define([
     Tabs,
     Toggles,
     isArray,
-    map,
     pick,
-    size,
     merge,
-    pairs,
-    chain
+    map,
+    reduce
 ) {
+    var urlBuilders = {
+        bestbuy:        defaultUrlBuilder('money/bestbuys'),
+        soulmates:      defaultUrlBuilder('soulmates/mixed'),
+        soulmatesTest:  defaultUrlBuilder('soulmates-test/mixed'),
+        capiSingle:     defaultUrlBuilder('capi-single'),
+        capi:           defaultUrlBuilder('capi'),
+        paidforCard:    defaultUrlBuilder('paid'),
```

Comment:
Let's order this:

defaultUrlBuilder,
complexUrlBuilder,
bookUrlBuilder,
soulmatesGroupUrlBuilder